iml-gui/configs/webpack.config.js

```diff
@@ -73,7 +73,7 @@ module.exports = (env, argv) => {
           },
           {
             from: "favicons",
-            to: "",
+            to: "whamcloud-favicons",
```

Comment:
Can we change this to whamcloud? Eventually we should move the logos out of the binary as well.